README.md

```diff
@@ -1,6 +1,17 @@
 API Application for camptocamp.org v6
 =====================================
 
+Requirements
+------------
+
+ * Python 3.4
+ * PostgreSQL 9.4 / PostGIS 2.1
+ * virtualenv (1.7 or higher)
```

Comment:
OK I will open a ticket for the sysadmins.
I don't know if it is related but when I try to run the indexing on my instance on the dev server, the load skyrockets (I have succeeded to index the main DB though...).